web/themes/new_weather_theme/assets/sass/components/_radar.scss

```diff
@@ -17,3 +17,45 @@
   border: 3px solid white;
   border-radius: 100%;
 }
+
+.wx-radar-legend {
+  h4.visual-h3 {
+    @include u-text("normal");
+
+    margin-top: 0;
+  }
+
+  .wx-radar-legend__row {
+    th {
+      vertical-align: top;
+    }
+
+    td {
+      padding: 0;
+      vertical-align: top;
+
```

Comment:
The top and bottom borders of the cells should be `color("base-light")` (currently they are the USWDS default)

Reply:
👍 I moved the border color and vertical alignment into a `th, td` selector instead of duplicating them, which is why this comment doesn't appear as outdated.